src/Crypto/Cryptograph.php

```diff
@@ -15,11 +15,11 @@ class Cryptograph
     /**
      * Constructor.
      *
-     * @param int $length
+     * @param int|null $length
      */
-    public function __construct($length = null)
+    public function __construct(int $length = null)
     {
-        $this->length = $length ?: 0;
+        $this->length = $length ?? 0;
```

Comment:
From a security point of view, ain't it better to set the default length to a higher number then 0?